tests/testthat/test_experiment.R

```diff
@@ -124,4 +124,42 @@ test_that("Create an interactive run, log metrics locally.", {
   files <- get_run_file_names(run)
   image_found <- grep("myplot", files)
   expect_true(length(image_found) > 0)
+})
+
+test_that("Create and submit child runs", {
+  skip_if_no_subscription()
+  ws <- existing_ws
+  
+  # start a remote job and get the run, wait for it to finish
+  tmp_dir_name <- file.path(tempdir(), "tmp_dir")
+  script_name <- "train_dummy.R"
+  dir.create(tmp_dir_name)
+  file.copy(script_name, tmp_dir_name)
+  
+  env <- r_environment("myenv", cran_packages = c("dplyr", "ggplot2"))
+  
+  est <- estimator(tmp_dir_name,
+                   compute_target = existing_compute$name, 
+                   entry_script = script_name,
+                   environment = env)
+  
+  exp <- experiment(ws, "estimator_run")
+  run <- submit_experiment(exp, est)
```

Comment:
we can create a parent run using iterative logging api and it returns immediately. i forgot the name of the function, it is called starting_logging or something.

Reply:
I found it! Thanks so much!